### PR TITLE
update antpool.php

### PR DESCRIPTION
**Pull request by xslugx**
add option for different coins instead of default BTC

### CODE REVIEW

File: classes/antpool.php [hunk 1]
```diff
@@ -38,12 +38,15 @@ function get($type) {
 		$nonce = time();
 		$hmac_message = $this->username.$this->api_key.$nonce;
 		$hmac = strtoupper(hash_hmac('sha256', $hmac_message, $this->api_secret, false));
+		// Set Coin Type: BTC, LTC, ETH, ZEC
+		$coin = '';
```
Comment (Elompenta) [inline classes/antpool.php]: see my comment in pull request

**Comment by Elompenta** [comment]
$coin is not a global variable so it doesnt work.
I think it is better if we change

`function get($type) {`

to 

`function get($type, $currency) {`

so you are able to request more than one currency with one instance of  the antpool api.

**Comment by xslugx** [comment]
Ok perfect, I wasn't sure how to do that specifically, but on using your code I had to figure out the implementation so that works for me

**Comment by Elompenta** [comment]
your version is fair enough for one setup, and the ide ais good!
I will check it, an provide a full functionally version with multi currency support.

thanks for your input

**Comment by xslugx** [comment]
I need to make a change to the custom.php due to bootstrap, I need to add flex to make it work with mobile better

**Comment by Elompenta** [comment]
@xslugx 
>I need to make a change to the custom.php due to bootstrap, I need to add flex to make it work with mobile better

Do it! ;)
Feel free to improve this class, and create new pull requests :-}
I will review your changes and commit them to the master branch.

thanks so far

File: classes/antpool.php (hunk 1 above)
Comment (Elompenta) [inline classes/antpool.php:42]: $coin is not a global variable so it doesnt work.
 I think it is better if we change
function get($type) {
to
function get($type, $currency) {
so you are able to request more than one currency with one instance of the antpool api.

**Review by Elompenta — CHANGES_REQUESTED** [review]
very nice idea - but currency hardcoded in class is not agile enough.

**Comment by Elompenta** [comment]
nice - i have already commited some minutes before a new Version ;)
Commit: https://github.com/Elompenta/antpool-php-api/commit/4bee623ace095a24422f5eea60f972746640ae34

I have integraed your changes and updated ssl version and readme
i will merge them to the master trunk

@xslugx also you are listed in the readme as contibutor :_}

**Review by Elompenta — APPROVED** [review]

**Comment by Elompenta** [comment]
changes already commited via branch pr/9

**Pull request closed without merging**